charts/ontology/README.md

```diff
@@ -0,0 +1,79 @@
+DUOS Ontology
+========
+A Helm chart for DUOS Ontology, the DUOS Algorithmic Matching System
+
+Current chart version is `0.1.0`
+
+
+
+
+
+## Chart Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| global.applicationVersion | string | Is set by Helmfile on deploy | Ontology global version |
+| appVersion | string | Is set by Helmfile on deploy | Ontology image version/tag. Required unless using `image`. |
+| devDeploy | bool | `false` | Enable to deploy to dev locally with Skaffold |
+| elasticSearch.server1 | string | `nil` | first elastic search host |
+| elasticSearch.server2 | string | `nil` | second elastic search host |
+| elasticSearch.server3 | string | `nil` | third elastic search host |
+| environment | string | `nil` | Environment, [dev, staging, prod] |
+| gcsAccount.path | string | `nil` | Vault path to GCS base64 service account json |
+| gcsAccount.key | string | `nil` | Vault key of GCS base64 service account json |
+| google.bucket | string | `nil` | Google project where GCS files are stored |
+| google.subdirectory | string | `nil` | Google bucket subdirectory |
+| imageConfig.repository | string | `nil` | GCR image location |
+| imageConfig.tag | string | `nil` | GCR image tag |
+| proxy.logLevel | string | `"debug"` | Proxy log level |
+| proxy.image.repository | string | `"broadinstitute/openidc-proxy"` | Proxy image repository |
+| proxy.image.version | string | `"bernick_tcell"` | Proxy image tag |
+| sentry.dsn.path | string | `nil` | Vault path to secret containing Sentry DSN value |
+| sentry.dsn.key | string | `nil` | Vault key of secret containing Sentry DSN value |
+| vault.enabled | bool | `true` | When enabled, syncs required secrets from Vault |
+| vault.pathPrefix | string | `nil` | Vault path prefix for secrets. Required if vault.enabled. |
+| vaultCert.enabled | bool | `false` | Enable to sync certificate secret from Vault with secrets-manager |
+| vaultCert.cert.path | string | `nil` | Path to secret containing .crt |
+| vaultCert.cert.secretKey | string | `nil` | Key in secret containing .crt |
+| vaultCert.key.path | string | `nil` | Path to secret containing .key |
+| vaultCert.key.secretKey | string | `nil` | Key in secret containing .key |
+| vaultCert.chain.path | string | `nil` | Path to secret containing intermediate .crt |
+| vaultCert.chain.secretKey | string | `nil` | Key in secret containing intermediate .crt |
+
+## Cheat Sheet
```

Comment:
This documentation is excellent!

Reply:
Thank you! I have cleaned it up a little using the helm-docs tool, very helpful!